src/coreComponents/finiteVolume/HybridFVMInnerProduct.hpp

```diff
@@ -302,6 +306,25 @@ struct QTPFACellInnerProductKernel
     LvArray::tensorOps::plusAikBkj< NF, NF, NF >( transMatrix,
                                                   worka_numFacesByNumFaces,
                                                   workc_numFacesByNumFaces );
+
+    // TODO: implement correctly the transmissibility calculation with multipliers
+    //
+    // we would like to do (if not too expensive):
+    // T = LvArray::tensorOps::invert< NF >( LvArray::tensorOps::invert< NF >( transMatrix ) + D )
+    // where D is diagonal matrix computed from the multipliers
+    //
+    // Ref: Nilsen, H. M., J. R. Natvig, and K.-A Lie.,
+    // "Accurate modeling of faults by multipoint, mimetic, and mixed methods." SPEJ
+
+    // placeholder before we can implement the hybrid treatment of trans multipliers
+    for( localIndex i = 0; i < NF; ++i )
+    {
+      real64 const mult = transMultiplier[elemToFaces[i]];
+      for( localIndex j = 0; j < NF; ++j )
+      {
+        transMatrix[i][j] *= mult;
+      }
+    }
```

Comment:
@corbett5 Here what I would like to ultimately do is to implement correctly the transmissibility multipliers for the hybrid FVM scheme. For this, I would need to compute the inverse of `transMatrix` in the kernel to do something like:
```
transMatrix = inv( inv( transMatrix ) + D )
```
where `transMatrix` is the local transmissibility matrix of size `NF x NF` and `D` is a diagonal matrix of size `NF x NF` involving the multipliers. For this, I would need `LvArray::tensorOps::invert< NF >` for `NF = 4`, `NF = 5`, and `NF = 6` (and maybe for more faces later). Based on what I saw in `LvArray/src/fixedSizeSquareMatrixOpsImpl.hpp` I think this is not supported yet but if you ever implement it I think this would be a possible use case (it is very low priority).

Reply:
Cool! As much as I would like to I don't have time to work on this, could you make an issue in LvArray so we can keep track of the request? Adding a general matrix inversion would be a departure from the current supported methods which are all closed form but that doesn't mean it's not a good idea. If you know of an algorithm that might work nicely for small sizes add it to the issue. Or if you think `(A^-1 + D)^-1` is a common pattern there might be a better way to specifically calculate that.

Reply:
Thinking about it more, I think I can use a workaround to compute `(A^-1 + D)^-1` without taking any inverse. Here, `A^-1` is a symmetric definite positive matrix, and `D` is a diagonal matrix. I can write this as `(A^-1 + D_1 + ... + D_{NF})^-1` where `D_i` are rank-one matrices. Then I can apply the Sherman-Morrison formula multiple times (for example, four times if `NF = 4`) to arrive at the result with matrix products and matrix sums only, without taking any inverse (I need to try to convince myself). 

I am going to give it a try and I'll let you know. This may be very specific to my problem though. If I don't arrive at anything satisfactory I will post an issue in LvArray. Thanks!

Reply:
Nice! I did a little toying around it seems like it should work. It would still have complexity of `n^3` which is the same as standard inversion but it is *probably* faster.

Reply:
@francoishamon There are a lot of zeros in there since the perturbation vectors are equal. And of course T is symmetric. It will be certainly faster than generic inversion. I think you perhaps make further simplifications aside from the obvious `MatrixMatrixMatrix` and `VectorMatrixVector products`. I can clean up the Mathematica sheet and send it with you if you would like to pursue it.

Reply:
The `VectorMatrixVector` product in the denominator works out to just a single multiplication for one iteration. The `MatrixMatrixMatrix` product in the numerator works out to O(N^2) multiplications.

Reply:
@rrsettgast Yes definitely that would be great if you could send it to me and I will implement it in this PR.

Thank you @rrsettgast @corbett5 for the help.

Reply:
@francoishamon I think I was chasing a ghost. I think the direct application of Sherman-Morrison taking advantage of the fact that `T(i)` is symmetric, `Outer( D(i), D(i) )` has a single nonzero value, and `D(i) T(i-1) D(i)` is a single FMA operation is the best you will do.

Reply:
I think it would look something like:
```suggestion
    // placeholder before we can implement the hybrid treatment of trans multipliers
    for( localIndex k = 0; k < NF; ++k )
    {
      real64 const mult = transMultiplier[elemToFaces[i]];
      real64 Td[NF];
      for( localIndex i = 0; i < NF; ++i )
      {
        Td[i] = T[k][i] * mult;
      }
      real64 const invDenom = 1.0 / ( 1.0 + Td[k] * mult );
      for( localIndex i = 0; i < NF; ++i )
      {
         for( localIndex j = 0; j < NF; ++j )
        {
          transMatrix[i][j] += Td[i]*Td[j]*invDenom ;
        }
     }
   }
```

Reply:
@rrsettgast Thank you for the suggestion it works great. To compute the `(A^-1 + D)^-1` (with a diagonal `D` involving the TPFA transmissibility and the multiplier), the formula does exactly what I need.